examples/awskinesisstream_scaledobject.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
```

Comment:
You can remove this, we no longer use these here.

Reply:
Removing other ones via https://github.com/kedacore/keda/pull/527

Reply:
thanks! I've removed this.